ts/src/sourcemapper/README.google

```diff
@@ -0,0 +1,13 @@
+URL:https://github.com/googleapis/cloud-debug-nodejs/blob/7bdc2f1f62a3b45b7b53ea79f9444c8ed50e138b
+Version: 7bdc2f1f62a3b45b7b53ea79f9444c8ed50e138b
```

Comment:
Is this brought via git subtree? If not, is there a practical need behind the README.google file?

Reply:
I removed the readme, and added a comment indicating where this file is originally from.
This isn't imported as a subtree (since it'll be modify enough that having a subtree probably won't be useful).